components/sapling_ai/package.json

```
@@ -1,17 +1,19 @@
 {
   "name": "@pipedream/sapling_ai",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

```suggestion
  "version": "0.1.0",
```
As per the versioning guidelines, new (backwards-compatible) features should warrant a new `minor` version (not a `patch` version)